src/components/FormInput/FormInput.test.tsx

```diff
@@ -0,0 +1,88 @@
+import userEvent from '@testing-library/user-event';
+import { useState } from 'react';
+import { describe, it, expect, vi } from 'vitest';
+
+import { render, screen } from '../../test/test-utils';
+
+import { FormInput } from './FormInput';
+
+describe('FormInput', () => {
+  const defaultProps = {
+    label: 'Mail Id',
+    value: '',
+    onChange: vi.fn(),
+    'data-testid': 'form-input',
+  };
+
+  const FormInputWrapper = ({ value: initialValue = '', onChange, ...props }) => {
+    const [value, setValue] = useState(initialValue);
+    const handleChange = (e) => {
+      setValue(e.target.value);
+      if (onChange) onChange(e);
+    };
+    return <FormInput {...props} value={value} onChange={handleChange} />;
+  };
+
+  it('renderiza con props por defecto', () => {
+    render(<FormInputWrapper {...defaultProps} />);
+    const inputContainer = screen.getByTestId('form-input');
+    expect(inputContainer).toBeInTheDocument();
+    expect(screen.getByLabelText('Mail Id')).toBeInTheDocument();
+    expect(screen.getByRole('textbox')).toHaveValue('');
+  });
+
+  it('renderiza con placeholder y value correctos', () => {
```

Comment:
Another test description is in Spanish ('renderiza con placeholder y value correctos'). Tests should use English descriptions per project rules.

**Sugerencia:**
```typescript
Translate to English, eg: it('renders with correct placeholder and value', () => { ... });
```